src/main/java/core/basesyntax/strategy/StorageReportGenerateImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class StorageReportGenerateImpl implements StorageReportGenerate {
+    public String getReport() {
```

Comment:
According to the checklist, empty lines at the beginning of class or method definitions should be removed.